scripts/build/fiwarelab/build-fiwarelab-image.sh

```diff
@@ -133,7 +133,7 @@ FIWARE_REPO=$(mktemp)
 cat > $FIWARE_REPO <<EOF
 [testbed-fiware]
```

Comment:
Just an aesthetical issue: we prefer calling it simply `[fiware]`, given that now repositories are not exclusive of the testbed enviroment.